scripts/run-benchmarks.main.kts

```diff
@@ -26,7 +32,7 @@ import okio.Buffer
 import okio.buffer
 import okio.source
 import java.io.File
-import java.util.*
+import java.util.Date
```

Comment:
For the record, [Roman Elizarov recommends star imports](https://youtrack.jetbrains.com/issue/KTIJ-14988/Force-star-import-for-a-package)